bitstream/src/bitread.rs

```diff
@@ -82,7 +82,9 @@ pub trait BitRead<'a>: BitReadInternal + Copy {
         if self.left() < n {
             n -= self.left();
             left = self.left();
-            ret = self.get_val(left);
+            if left > 0 {
```

Comment:
is it really needed?

Reply:
Allowing n=0 for peek_val() as discussed removes the need for it